sdk/servicebus/service-bus/test/utils/testUtils.ts

```diff
@@ -43,11 +43,11 @@ export class TestMessage {
       contentType: `content type ${randomNumber}`,
       correlationId: `correlation id ${randomNumber}`,
       timeToLive: 60 * 60 * 24,
-      label: `label ${randomNumber}`,
+      subject: `label ${randomNumber}`,
```

Comment:
@mohsin-mehmood 
Can you also add `userId` field in this? ..since that would help cover the attribute in the tests.